packages/core/src/schema/models/commits.ts

```diff
@@ -25,6 +25,9 @@ export const commits = latitudeSchema.table(
     projectId: bigint('project_id', { mode: 'number' })
       .notNull()
       .references(() => projects.id, { onDelete: 'cascade' }),
+    userId: text('user_id')
```

Comment:
I think we are calling it author_id/authorId in other places maybe we wanna maintain consistency

Reply:
Is it super important? I would rather not make this change.